build.override.targets

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0"  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--if ServiceUri is set, the service has been already started on another machine-->
```

Comment:
That is up to you guys but do keep in mind it isn't just about sharing it is about readability as well as the conditions are very log right now and I cannot even see the full line without scrolling.